ft_lib/ft_upper.py

```diff
@@ -0,0 +1,14 @@
+
+def ft_upper(obj):
+    l = ''
+    if isinstance(obj, str):
+        for i in obj:
+            if ord(i) in range(97, 123):
+                j = ord(i) - 32
+                l = l + chr(j)
```

Comment:
you can write something like that `l = l + chr(ord(i) - 32)` and remove `j = ord(i) - 32`